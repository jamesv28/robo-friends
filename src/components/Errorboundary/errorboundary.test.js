import {shallow} from 'enzyme';
import ErrorBoundary from './errorboundary.component';
import React from 'react';

it('should take a look at snapshot testing with error boundary', () => {
    expect(shallow(<ErrorBoundary />)).toMatchSnapshot();
})