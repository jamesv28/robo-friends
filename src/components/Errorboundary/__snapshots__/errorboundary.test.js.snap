// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a look at snapshot testing with error boundary 1`] = `ShallowWrapper {}`;
